Jenkinsfile

```diff
@@ -414,11 +414,11 @@ pipeline {
       # Every weeknight (Monday - Friday) around 2:00 AM
       ### JDK8 tests against 2.1, 3.0, DSE 4.8, DSE 5.0, DSE 5.1, dse-6.0.18 and DSE 6.7
       H 2 * * 1-5 %CI_SCHEDULE=WEEKNIGHTS;CI_SCHEDULE_SERVER_VERSIONS=2.1 3.0 dse-4.8.16 dse-5.0.15 dse-5.1.35 dse-6.0.18 dse-6.7.17;CI_SCHEDULE_JABBA_VERSION=1.8
-      ### JDK11 tests against 3.11, 4.0 and DSE 6.8
-      H 2 * * 1-5 %CI_SCHEDULE=WEEKNIGHTS;CI_SCHEDULE_SERVER_VERSIONS=3.11 4.0 dse-6.8.30;CI_SCHEDULE_JABBA_VERSION=openjdk@1.11
+      ### JDK11 tests against 3.11, 4.0, 5.0-beta1 and DSE 6.8
+      H 2 * * 1-5 %CI_SCHEDULE=WEEKNIGHTS;CI_SCHEDULE_SERVER_VERSIONS=3.11 4.0 5.0-beta1 dse-6.8.30;CI_SCHEDULE_JABBA_VERSION=openjdk@1.11
       # Every weekend (Sunday) around 12:00 PM noon
-      ### JDK14 tests against 3.11, 4.0 and DSE 6.8
-      H 12 * * 0 %CI_SCHEDULE=WEEKENDS;CI_SCHEDULE_SERVER_VERSIONS=3.11 4.0 dse-6.8.30;CI_SCHEDULE_JABBA_VERSION=openjdk@1.14
+      ### JDK14 tests against 3.11, 4.0, 5.0-beta1 and DSE 6.8
+      H 12 * * 0 %CI_SCHEDULE=WEEKENDS;CI_SCHEDULE_SERVER_VERSIONS=3.11 4.0 5.0-beta1 dse-6.8.30;CI_SCHEDULE_JABBA_VERSION=openjdk@1.14
```

Comment:
This makes less sense to me.  There's nothing magic about Java14.  My guess is this was the most recent Java release when this change was made and it just hasn't been updated.

It only seems to make sense to run nightlies against current LTS releases so I'd change this from Java14 to Java17 as well.